speech/unit_tests/test__gax.py

```diff
@@ -39,11 +38,38 @@ def test_ctor(self):
         from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
             SpeechContext)
         from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+            RecognitionConfig)
+        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
+            StreamingRecognitionConfig)
+        from google.cloud.grpc.speech.v1beta1.cloud_speech_pb2 import (
             StreamingRecognizeRequest)
 
-        sample = Sample(source_uri=self.AUDIO_SOURCE_URI,
+        sample = Sample(content=self.AUDIO_CONTENT,
                         encoding=speech.Encoding.FLAC,
                         sample_rate=self.SAMPLE_RATE)
-        streaming_request = self._callFUT(sample, 'US-en', 2, True,
-                                          SpeechContext(), True, True)
+        language_code = 'US-en'
+        max_alternatives = 2
+        profanity_filter = True
+        speech_context = SpeechContext(phrases=self.HINTS)
+        single_utterance = True
+        interim_results = False
+
+        streaming_request = self._callFUT(sample, language_code,
+                                          max_alternatives, profanity_filter,
+                                          speech_context, single_utterance,
+                                          interim_results)
         self.assertIsInstance(streaming_request, StreamingRecognizeRequest)
+        self.assertIsInstance(streaming_request.audio_content, bytes)
```

Comment:
So at this point, `streaming_request.audio_content` should be empty `b''`. The first `StreamingRecognizeRequest` **must** not have `audio_content` set and only `streaming_config` set.
The second `StreamingRecognizeRequest` can only have `audio_content` set and not `streaming_config.

I added a comment about this in the test for now.